core/store/src/db.rs

```diff
@@ -83,8 +83,8 @@ pub enum DBCol {
     /// `LightClientBlock`s corresponding to the last final block of each completed epoch
     ColEpochLightClientBlocks = 26,
     ColReceiptIdToShardId = 27,
-    ColNextBlockWithNewChunk = 28,
-    ColLastBlockWithNewChunk = 29,
```

Comment:
I'd expect to see a comment explaining `_`

Reply:
Hmm actually just took the existing code `_ColTransactionRefCount` as an example. I am not sure what's the right way to delete/deprecate a column from the database

Reply:
Don't think we have a process for that 😅 Feel free to propose something!

Reply:
I did a few things:
1) moved all the deprecated columns to the end of enum so it's easier to see all of them
2) added a database migration to remove data from deprecated columns

I think for now I am just going to keep the deprecated columns still in the database and in the codebase. We may want to drop the column family entirely, but I think that should be in another PR because it's relatively involved and I don't have time right now. I can open an issue for that if you think it's necessary.

Reply:
I actually revert the change on 2) because some test were broken and I couldn't quickly figure out why. I don't want to spend too much time on this PR right now so I'll just create an issue to track this